packages/react/src/components/Authenticator/Authenticator.tsx

```diff
@@ -20,6 +20,22 @@ export type AuthenticatorProps = AuthenticatorMachineOptions &
   RouterProps &
   ComponentsProviderProps;
 
+// Helper component that sends init event to the parent provider
+function InitMachine({ children, ...machineProps }) {
+  const { _send, route } = useAuthenticator();
+  React.useEffect(() => {
+    if (route === 'idle') {
+      _send({
+        type: 'INIT',
+        data: machineProps,
+      });
+    }
+    // Disabling exhaustive hooks rule here because we only want to run a single time
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Prefer passing `route` and `machineProps` to the deps array of the `useEffect`, the conditional on ln 27 will prevent the call to `_send` from executing over ignoring the **eslint** rule. 

Assuming there is potential of further calls to `_send` triggering when `route` updates to `'idle'` after the initialization call we can gate with a ref assigned variable (e.g. `hasInitialized`) to ensure that the call to `_send` only happens once. Thoughts?